### PR TITLE
Removes several round-start disabilities, adds chav.

### PR DESCRIPTION
**Pull request by Shadeykins**
**What does this PR do:**
_This PR will probably require wiping the saved disability preferences for characters from the SQL database._

Removes Epilepsy, Tourette's, and Scrambled (Can't speak properly) from character disability preferences while adding Chav as an option. This does not remove the ability to get these disabilities from genetics, only the capacity to spawn in with them on round-start.

These disabilities (while occasionally hilarious) add little to the round, create headaches for other players who have to put up with epileptic Tourette's ridden nuggets, and are rarely used in any capacity that genuinely benefits the server. They are, in almost all cases, used in a way which runs directly counter to the server's rules about roleplay:

> Paradise Station is a Medium Roleplaying Server. This means that we do not expect players to completely and absolutely immerse themselves in the universe of the game. However, we still expect players to act in-character at all times, **in a manner befitting the overall scenario: that of working aboard a high-tech research space station owned by a Galactic MegaCorp with dubious moral standards** (emphasis mine).

This will also resolve a running issue w/ important roles (medical, engineering, security) having disabilities which sincerely impair their ability to function.

I think I appropriately caught all relevant cases for prefs/restrictions. Please let me know if I missed something/broke all the things.

**Changelog:**
:cl:
add: Added Chav to roundstart character preferences.
del: Removed Epilepsy, Tourette's, and Can't speak properly from roundstart character preferences.
/:cl:

### CODE REVIEW

**Comment by KasparoVy** [comment]
Deafness is where I draw the line as it does not have the spammy/memey nature of the other three disabilities and is one I take on occasion in noncritical roles.

I disagree (matter of taste) with the removals wholesale although agree with the points you made and feel it has the potential to enact the desired effect.

**Comment by Shadeykins** [comment]
> 
> 
> Deafness is where I draw the line as it does not have the spammy/memey nature of the other three disabilities and is one I take on occasion in noncritical roles.
> 
> I disagree (matter of taste) with the removals wholesale although agree with the points you made and feel it has the potential to enact the desired effect.

Thanks. I included deafness because it effectively disables communication with players. Mute people can still be spoken at, deaf players can't. Hopefully that clarifies why it's in there.

**Comment by KasparoVy** [comment]
I understand and appreciate the explanation but would still like to remain respectfully deaf (on occasion). I wonder if it would be best to re-evaluate departmental disability restriction like is present for Captain (and I think some other command roles).

**Comment by Twinmold93** [comment]
My desire would be to see it removed from important roles, though honestly per the RP standards desired and mentioned, some of these shouldn't exist as available to standard crew. They do make the roles very hard to work with. Deafness, I feel, could stay, though again not something an important verbal (high-text) role should have. Generally, it makes communications very difficult without hoping you have paper and pen (people use those) or a PDA.

**Comment by KasparoVy** [comment]
I feel that conversation would be better to have than outright removal which in effect is a more severe course of action.

**Comment by Shadeykins** [comment]
> 
> 
> I feel that conversation would be better to have than outright removal which in effect is a more severe course of action.

#8292

**Comment by Allfd** [comment]
In general the team is a no on whitelists of any kind that require applications.  Technically, if you convinced someone nicely you had a RP/Valid round start reason that would be one thing.

The problem is, for the most part this is just for memes.

**Comment by DrunkDwarf** [comment]
None of these truly run counter to the rules for roleplay, if anything this change reduces roleplay in favour of the powergame meta of doing everything the most efficient way possible.

None of these disabilities would prevent you completing most jobs on the station, even deaf. At worst, Deaf just makes is "less optimal" due to the restrictions of the game engine, such as other forms of communication being lacking. In real life, they are rarely a problem, apart from very specific careers.

I can see an argument being made for restrictions for command, as has been attempted in the past, but this just removes the option for anyone even in unimportant roles having the ability to roleplay these disabilities. This is absurd.

This smells of either being annoyed at people disrupting your powergaming, or having a severe misunderstanding/prejudice toward how severe these disabilities are in real life.

Arguments about it causing headaches dealing with them, or that it's played only for memes, could be applied to 99% of the game, so why pick these out specifically?

**Comment by Shadeykins** [comment]
> 
> I can see an argument being made for restrictions for command, as has been attempted in the past, but this just removes the option for anyone, even in unimportant roles, having the ability to roleplay these disabilities. This is absurd.
> 
> This smells of either being annoyed at people disrupting your powergaming, or having a severe misunderstanding/prejudice toward these disabilities in real life.
> 
> Arguments about it causing headaches dealing with them, or that it's played only for memes, could be applied to 99% of the game, so why pick these out specifically?

You're aware that these disabilities exist in-game as jokes/memes, correct? People with epilepsy do not actually have seizures every two minutes, nor do people with speech impediments drop all clauses and prepositions while screaming in two word sentences at the top of their lungs.

If any of these were reasonably implemented (which is beyond the scope of this PR), this would not be up. They are not, mostly because they were designed with balancing radiation/genetics and fun in-mind (rather than roleplay). Further, I would appreciate it if you refrained from implying that I'm a bigot for not wanting people to spawn in with shitpost-tier implementations of epilepsy.

**Comment by Spacemanspark** [comment]
You can always role play having said disability too, you know. You don't need the system to hold your hand for roleplay.

**Comment by Citinited** [comment]
I None of these truly run counter to the rules for roleplay as they're currently written (regardless of how you cherry pick), if anything this change reduces roleplay in favour of the powergame meta of doing everything the most efficient way possible.>
The default position is one with no disabilities. Someone not picking disabilities (and assuming other characters don't have them) is not "powergame meta", it's not voluntarily nerfing your character.

Also, IMO it's not powergaming to not want to have to jump through hoops to accomodate someone else's voluntarily chosen debuff. This isn't IRL, nobody is forced to be deaf or mute or whatever.

**Comment by Citinited** [comment]
> None of these truly run counter to the rules for roleplay as they're currently written (regardless of how you cherry pick), if anything this change reduces roleplay in favour of the powergame meta of doing everything the most efficient way possible.

The default position is one with no disabilities. Someone not picking disabilities (and assuming other characters don't have them) is not "powergame meta"; instead it's not voluntarily nerfing your character.

Also, IMO it's not powergaming to not want to have to jump through hoops to accomodate someone else's voluntarily chosen debuff. This isn't real life; nobody is forced to be deaf or mute or whatever. It's just another way you choose to play a character, and choices have repercussions.

**Comment by DrunkDwarf** [comment]
Most of the game is composed of jokes/memes that aren't conducive to running an optimal science research station, if efficiency and preventing annoyances to the crew is suddenly more important, then why even have Clowns?

You can roleplay a lot of the game without needing the mechanics, yet the mechanics exist to make life easier.

If the implementations are poor, then **improve instead of remove**. 

Deaf: This is not poorly implemented, is not even joke/meme level. Even if it reduces communication, how does it make any difference to non-command/sec roles? Many roles on the station can be, and are, done without the slightest bit of communication already. PDA messaging is already deaf-friendly, it's just not the preferred form of communication. 

Epilepsy: You could reduce the frequency of seizures to be more realistic, an easy fix. Random seizures outside of the important roles have no bearing on the round at all. True, you'd not want your surgeon having seizures mid operation, but nobody cares if the Cargo Tech is prone on the floor on occasion. This should not be removed from all players.

Tourettes: This could also just be toned down as an alternative to removal, if the effects are too disruptive. Again, you should still be able to communicate with this, and should not be removed for all players.

IMO, this PR should be changed to disallowing it for important roles specifically, rather than trying to get that same result by removing it for everyone.

**Comment by Citinited** [comment]
Improve don't remove meme needs to die 😄

**Comment by MINIMAN10000** [comment]
Although here I am thinking if implemented properly how would they all be?
Deaf would stay the same
Epilepsy would require flashing light sources IE flashing a lightswitch.
Tourettes, I mean in reality it seems to show up as just a tick, really don't see a need to implement.
Scrambled... "but why?"
So in the end the only ones I can see really being implmented are dead and epilepsy the others just seem to exist for laughs.
Ultimately I've never ran into them in game nor do I use them myself so I don't particularly care.

**Comment by Shadeykins** [comment]
> 
> IMO, this PR should be changed to disallowing it for important roles specifically, rather than trying to get that same result by removing it for everyone.

#8292

Conversation had, role restrictions not happening.

Want to improve it? Make a PR.

That will solve the concerns of this PR as well as your own, but I am not going to do that for you.

**Comment by DrunkDwarf** [comment]
> #8292
> 
> Conversation had, role restrictions not happening.

And yet this PR is trying to add some of the same restrictions as that PR, but making it even broader to apply to all roles. Surely the same arguments against that PR also apply here.

**Comment by Shadeykins** [comment]
> 
> 
> > #8292
> > Conversation had, role restrictions not happening.
> 
> And yet this PR is trying to add some of the same restrictions as that PR originally was, but making it even broader to apply to all roles. Surely the same arguments against that PR also apply here.

This PR is not adding any role-based restrictions, it is simply removing the option to select these as a preference. A globally enforced rule is very different in practice (and far more straightforward) than having to go ahead and specify what disabilities which roles are allowed to have.

If you desire an alternative, I encourage you to make a PR.

**Comment by tylerh95** [comment]
Agreeing with improving instead of removing for all except for tourette's. Tourette's, in it's current implementation, is pretty memey.

However I've played with multiple players who use the others ones and roleplay well. If anything, the job blacklist for certain disabilities should be modified, rather than removing them from character preferences.

**Comment by FalseIncarnate** [comment]
In its current state, Epilepsy definitely seems a bit heavy handed in how it works. Aside from certain situations, someone with epilepsy can definitely perform a job without much issue or accommodation (when I worked at a restaurant, one of our servers had epilepsy. He could carry trays of food and interact with guests as well as anyone, but wasn't allowed to drive. In the 3 years I worked with him, I'm not sure he ever had a seizure at work). If we portrayed it more realistically (triggering with flashes or such), I'd have no problem leaving it as a round-start disability, but as-is it just exists to make your character bounce around randomly. I think this one should probably have the disability reworked if it is to remain, but removed from round-start until such changes are in place.

Deafness is definitely a block to communication, though able to be circumvented if you are aware of it, so I can definitely see that argument going either way on remaining round-start capable. That said, there are definitely jobs in real life that a deaf person would not be allowed or able to perform, but short of blocking deaf people from security and command we really don't have any jobs where it would actively inhibit your ability to perform the duties of a role. Since we really don't want to do whitelists or further role-restrictions, I'm fine with this getting dropped if we must.

Tourette's is honestly the least problematic of the three disabilities in my mind. Yes, it's kinda low-brow and meme material, but honestly by itself really doesn't hamper anything other than interrupting conversations with a random "TITS!" now and again. The only real issue is once it gets combined with things like epilepsy and missing limbs to create the infamous "nugget" characters that aren't really trying to roleplay much anyways. I am not particularly convinced this one should go, but also wouldn't be distraught in the slightest if it did.

TL;DR:
Epilepsy should get reworked, remove from roundstart disabilities until such time then re-add
Deafness impacts Security and Command hardest, and is fine to drop because we don't want to whitelist
Tourette's can stay or go, it doesn't really get in the way on its own but doesn't really add much either

**Comment by KasparoVy** [comment]
I've exhausted reason and see where this is going, I have nothing left to add but 'no u'

**Comment by Allfd** [comment]
Lots of people are saying "Instead of doing this, it should be reworked."  Is someone going to do that or is it a false choice?

Two options exist,  keep things as is, or this specific change in this PR.

If someone makes an alternative PR, then a choice exists, until then, it does not.

**Comment by Ralta** [comment]
There's an ungodly amount of false information in this about 'real life' disabilities. Epilepsy, for example, is not always triggered by light. It can be triggered by low blood sugar, for example. Or a hundred other things. But anyway...

I'm very on the fence about removing these disabilities as round-start options. As @Allfd says, we have two options currently presented: remove, or remain as we are. In that case, I would favour remaining as we are.

However, I would be in favour of adjustments to either the way those round-start disabilities work (which I appreciate might require some snowflaking to construct 'special' versions of the disabilities), or denying characters from taking specific roles with those disabilities. Say what you will about 'improve instead of remove', but I think it's a reasonable stance to take here.

There are arguments to be made about realism, and how you'd be in all kinds of trouble for not hiring someone based solely on a disability...except that's not accurate. You cannot get in trouble for choosing not to hire someone based on the fact that they have a disability if it would make them incapable of performing their job correctly. You would not hire a wheelchair-bound quadriplegic as a bouncer at a club. You also wouldn't hire a fully deaf individual as your Chief Engineer/Chief Medical Officer/Head of Security etc.

I do believe there is a problem to be solved here, but I don't think this PR is the right solution.

**Comment by Shadeykins** [comment]
@Ralta @tylerh95 If you have a better implementation, I highly encourage you to make a PR for it as we will all be better off for it. It's beyond the scope of this PR/my familiarity with BYOND spooky code and I will not be doing alternative implementation (whitelists/blacklists beyond Captain/HoS have already been given a hard no).

@KasparoVy If I'm correct, it sounds like you utilize deafness to "turn off" player speech or sounds. Could you go into a bit of detail on why you use deafness/what it provides that's useful to you?

**Comment by KasparoVy** [comment]
Certainly! 

After particularly aggravating rounds on the regular I'll use a deaf character (or simply tack it into an existing one/randomnamer) and go into a low-comms role like janitor, miner or what have you. It provides respite and reduces sodium content in my spess stream while still allowing me to play the game and RP, see below

I go deep full focus on my job with some music in the background and for important interactions I PDA, paper, go all in RP in pantomime or body language.
Quite a few entertaining and outright reinvigorating interactions happen because of the alternate comms alone.

Furthermore it means I'm quite (often pleasantly) surprised when shit goes down because I lack the ability to see stuff unfolding in comms. It keeps my experience unpredictable and in the moment through this wack method. I'm also more vulnerable to antags during this time since my ability to counter by way of audio-based cues is nullified.
I feel less mindful of meta business and end up dying more as a result, giving antags more of a chance--and I've accepted that risk. Sometimes I find myself too focused on the way things normally are/stuff I ICly shouldn't know and this helps keep it out of my head. 

It's a stark contrast to the way I normally play the game so this novel variation coupled with the low-impact jobs I occupy during use of the disability and the positive benefits (albeit quite heavily interpretive and circumstantial) make it something I'm happy about being in character select

**Comment by Shadeykins** [comment]
@KasparoVy Thanks, that seems like a definite usecase and something that actually adds value. I put deafness in because of its potential to shut off lines of communication (which I still think is an issue), though it is far less of an issue than the other three (which are far more in your face and exist in their implementation as crude jokes).

I have tentatively added it back in, but I may revert such a change if it ends up impacting the approval of the PR.

**Comment by Birdtalon** [comment]
"Improve don't remove" is a stupid argument. Features which don't fit should not have to be improved upon. Removal of content is essential for any ongoing project as it advances because not all content will forever be appropriate.

That being said I am downvoting this because I don't think removing these as round-start is really necessary. Memes and jokes have a place in this game.

**Comment by Pennwick** [comment]
I'm okay with some of these other disabilities staying but could we PLEASE remove the 50% chance of tourettes of broadcasting to the common channel. Its never funny. I thought it was just assholes switching their headset to always on but I double checked the code and its actually a random chance thing.

**Comment by Landerlow** [comment]
I do not agree with removal of any of the disabilities, however I would like for it to be impossible to have command roles with any disabilities enabled. These disabilities cause headaches because people fail to roleplay properly, it's not the disabilities themselves. It should be fairly obvious to people that one should not go into sec/command with any of these disabilities. 

I also do not agree with removing them because I put them in there to make things more interesting for players. You could simply add a rule/SoP/both saying "You are not allowed to play important roles with disabilities."

[Insert roles here]

I do not think content should be removed so fast because people fail to use it properly. Especially something relatively small as this. If someone has a command role with a disability, someone can ahelp it, and the admin can take care of it.

**Comment by KasparoVy** [comment]
Sad, but I'm glad deafness was spared.
